java/src/main/java/com/google/cloud/dataproc/templates/hbase/README.md

```diff
@@ -0,0 +1,94 @@
+## 1. HBase To BigQuery
+### Required JAR files
+
+Some HBase dependencies are required to be passed when submitting the job. These dependencies are automatically set by script when CATALOG environment variable is set for hbase table configuration. If not, 
+these dependencies need to be passed by using the --jars flag, or, in the case of Dataproc Templates, using the JARS environment variable. 
+- **[Apache HBase Spark Connector](https://mvnrepository.com/artifact/org.apache.hbase.connectors.spark/hbase-spark) dependencies (already mounted in Dataproc Serverless, so you refer to them using file://):**
+    - file:///usr/lib/spark/external/hbase-spark-protocol-shaded.jar
```

Comment:
Given now we are importing jars do we still need this in readme ?

Reply:
We can remove that but I thought this can be a good add on information hence added. Happy to remove it if it doesn't fit well.